# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-ia64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./src/include/override/sys/syscall.h.vanilla	2025-09-30 20:43:19.408505160 +0200
+++ ./src/include/override/sys/syscall.h	2025-09-30 20:45:12.874467198 +0200
@@ -276,7 +276,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__ia64__)
-#    define systemd_NR_open_tree_attr -1
+#    define systemd_NR_open_tree_attr 467
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__m68k__)
@@ -486,7 +486,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_removexattrat 466
 #  elif defined(__ia64__)
-#    define systemd_NR_removexattrat -1
+#    define systemd_NR_removexattrat 466
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_removexattrat 466
 #  elif defined(__m68k__)
@@ -556,7 +556,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_setxattrat 463
 #  elif defined(__ia64__)
-#    define systemd_NR_setxattrat -1
+#    define systemd_NR_setxattrat 463
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_setxattrat 463
 #  elif defined(__m68k__)
